### DISCUSSION TITLE
【要望】鉄道関係の記事追加

### DISCUSSION THREAD

**Discussion by graycat27** [freeserverproject/freeserver-wiki#114] [Ideas]
もともとホームページに載っていた、公営鉄道の案内および私鉄建設の案内に準ずる内容の記事がほしい
- 公営鉄道(FR線)の案内（路線図・中心町駅の乗り換え案内）
- 私鉄建設ルールの案内（Discordで申請するときの情報テンプレもほしいかも）
- 既存の私鉄の記事（GOサイン出れば、灰猫鉄道は @graycat27 が書きます）

**Comment by Nakatai-0322** [comment]
<https://github.com/freeserverproject/freeserver-wiki/pull/140>

FRに関する記事群、追加しました。

**Reply by Nakatai-0322** [reply]
@graycat27

**Reply by Nakatai-0322** [reply]
良ければレビューお願いします！
夜分遅く（かもしれない）に申し訳ない！
